_overviews/scala3-book/packaging-imports.md

```diff
@@ -81,12 +81,9 @@ They’re explained more in the subsections that follow.
 Import statements are also used to import `given` instances into scope.
 Those are discussed at the end of this chapter.
 
-Two notes before moving on:
+A note before moving on:
 
 - Import clauses are not required for accessing members of the same package.
```

Comment:
Now that it is only a single note, maybe drop `A note before moving on:` and  replace by:

```suggestion
> Import clauses are not required for accessing members of the same package.
```
?